envs/example/defaults-2.0.yml

```diff
@@ -207,6 +207,9 @@ nova:
       disk: 160
       ephemeral: 0
       vcpus: 8
+  service:
```

Comment:
We do not need new variables defined here. Instead, add to roles/openstack-meta/defaults/main.yml under service where we need to set env with cafile. Also, need to enable for rhosp and unit test on it as well.